components/webext-storage/src/lib.rs

```diff
@@ -42,3 +43,17 @@ impl UniffiCustomTypeConverter for JsonValue {
         obj.to_string()
     }
 }
+
+// Our UDL uses a `Guid` type.
```

Comment:
I wonder if we should just use `Guid` here and in the UDL? I'm not sure `WebExtStorageGuid` adds much value and it might even mislead people into thinking it's special when it's not.